proto/viam/app/package/v1/package.proto

```diff
@@ -0,0 +1,103 @@
+syntax = "proto3";
+
+package viam.app.package.v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+option go_package = "go.viam.com/api/app/package/v1";
+
+service PackageService {
+  // CreatePackage uploads a package to the cloud
+  rpc CreatePackage(stream CreatePackageRequest) returns (CreatePackageResponse) {
+    option (google.api.http) = {
+      post: "/viam/app/package/v1/create"
```

Comment:
@sbal13 Would be useful to get your feedback on these names for consistency with typical REST endpoints. 

@tahiyasalam originally suggested :organization_id here but there will always be an org, package name, and package version, which seems like too much to include :)

Reply:
Eric Adam and I talked about this and decided that the viam and app bits are unnecessary, though you'll see this format in other protos that were made before this. Basically we decided it should be `/<service>/<verison>/<action` so in this case just the `/package/v1/create` bit

Reply:
we're not necessarily going for REST conventions here, more just trying to make them expressive of the services/functions that these endpoints are proxies for

Reply:
cool!

Reply:
Thanks Steven - @tahiyasalam I think this all looks great except seems like can get rid of the /viam/app prefix!

Reply:
my bad! should be fixed now